style.css

```diff
@@ -401,3 +402,120 @@ h4 {
 .btn:focus {
   background-color: #008552;
 }
```

Comment:
- [ ]  Great job using the correct button type and text. 👍 I think there is a little improvement you need to make to the UI. The Figma design provides the colours for different states of the button `hover`, `enabled` and `pressed` and according to the project requirements, you are supposed to stick to these designs as much as you can. I found out that the `.btn:enabled` and `.btn:focus` classes has overridden the functionality of the `btn:hover` class. I highly recommend removing `.btn:enabled` and `.btn:focus` inorder for the `hover` effect functionality to work. Which means there is no need adding `.btn:enabled` and `.btn:focus` classes. 😄

![Screenshot (116)](https://user-images.githubusercontent.com/78104544/228458951-85fdc828-5184-4874-a0da-2b2df448d829.png)